lib/renderer.coffee

```diff
@@ -141,6 +141,15 @@ module.exports =
         cb null, result
       lang: -> 'html'
       exts: /\.(pug)$/i
+    'Handlebars':
+      render: (text, filepath, cb) ->
+        Handlebars = allowUnsafeNewFunction -> allowUnsafeEval -> require 'handlebars'
+        template = allowUnsafeNewFunction -> allowUnsafeEval ->
+          Handlebars.compile text
```

Comment:
@chaliy I searched the Handlebars API and can't find whether compile would throw on faulty syntax. Is this a case you need to account for?

Reply:
@chaliy are you still interested in looking into this?